tendrl/commons/tests/test_init.py

```diff
@@ -73,7 +73,8 @@ def test_list_modules_in_package_path():
         ('platform', 'tendrl.commons.objects.platform'),
         ('service', 'tendrl.commons.objects.service'),
         ('tendrl_context', 'tendrl.commons.objects.tendrl_context'),
-        ('block_device', 'tendrl.commons.objects.block_device')]
+        ('block_device', 'tendrl.commons.objects.block_device'),
+        ('virtual_disk', 'tendrl.commons.objects.virtual_disk')]
```

Comment:
I have already this change in -> https://github.com/Tendrl/commons/pull/559/files#diff-42379248bd3053683b4604f479e96dc2R78